tests/commands/retry_test.py

```diff
@@ -181,26 +187,3 @@ def test_wait_for_retry_deps_done(fake_retry_action, mock_client_request, event_
         data=dict(command="retry", use_latest_command=1),
         user_attribution=True,
     )
-
-
-@mock.patch.object(retry, "RetryAction", autospec=True)
-def test_retry_actions(mock_retry_action, mock_client, event_loop):
```

Comment:
I previously removed this for the validation-only Jammy tron version. I think this is relevant for an actual tron release

Reply:
I kept the test and it seems to be passing